src/csharp/Grpc.IntegrationTesting/ServicesGrpc.cs

```diff
@@ -47,12 +47,28 @@ public interface IBenchmarkServiceClient
     }
 
     // server-side interface
+    [System.Obsolete("Service implementations should inherit from the generated abstract base class instead.")]
     public interface IBenchmarkService
     {
       Task<global::Grpc.Testing.SimpleResponse> UnaryCall(global::Grpc.Testing.SimpleRequest request, ServerCallContext context);
       Task StreamingCall(IAsyncStreamReader<global::Grpc.Testing.SimpleRequest> requestStream, IServerStreamWriter<global::Grpc.Testing.SimpleResponse> responseStream, ServerCallContext context);
     }
 
+    // server-side abstract class
+    public abstract class BenchmarkServiceBase
```

Comment:
Having got my head round "this is the server side base, we're not providing an implementation" I'm in favour of it being called HealthBase, and I apologise for previous confusion.

That doesn't mean the change from Health to HealthGrpc is a bad idea - I quite like that.